### ISSUE TITLE
Feature Request: Decorate Server Sync-able and Sync-ed Settings

### ISSUE THREAD

**Issue by rdoll** [Lurkars/gloomhavensecretariat#404]
### Is your feature request related to a problem? Please describe.

Sync Settings is a great way to ensure everyone in our game has a similar _core_ experience in GHS, but all players in the same Gloomhaven/Frosthaven session can't have the _exact_ same experience without also using the same local settings. However, which settings need to be communicated to the group isn't clear.

### Describe the solution you'd like

Next to each server sync-able setting, use an icon to indicate that the setting can be (or is being) synced.

Example icons from the Noun project: 
![icon_2072921_edited](https://github.com/Lurkars/gloomhavensecretariat/assets/7015158/1db8cd87-2d95-49c9-8abd-aab3a2f2aa7a =100x100) ![icon_4338348_edited](https://github.com/Lurkars/gloomhavensecretariat/assets/7015158/8d3353f6-dfaf-4c00-ad3a-ddb4f2167ea2 =100x100)

If the client also has Sync Settings enabled, we'd color the icon green.

The server sync icon would have a tooltip for sync-able (e.g. can be automatically set from the server when Sync Settings is enabled) and sync-ed (e.g. is being automatically set from the server because Sync Settings is enabled).

### Describe alternatives you've considered

We could also add an icon for settings that are not server sync'ed, but that is the commonly understood default, so it seemed like overkill.

### Additional context

For implementation, we could add a list of local settings (because there are fewer local only settings) somewhere, maybe in the `Settings` model class. Then `settings.html` can easily use it to `*ngIf` the server sync icon and `SettingsManager.ts` can use it to control which server sync'ed settings get overwritten by the local settings. This also centralizes the demarcation of which settings are local and which can be server synced.

**Comment by Lurkars** [comment]
Good idea, should be a quick change. Just have to look if additional icons in settings doesn't look too bloated, I'll check.

**Comment by Lurkars** [comment]
I added icons to the headings in settings in [v0.77.9](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.77.9). Looked better to me than next to every settings and I just had to move one settings to the interface row that the icons are correct. (So only visible when `Synchronize Settings` is enabled, if connected icons should be green)

**Comment by rdoll** [comment]
Yup, this is perfect. I verified connected and disconnected with and without Sync Settings enabled. And thinking about it, yeah, putting it on the group makes perfect sense. Nice job and thanks!
